### ISSUE TITLE
Add ignore comments for auto generated files like sst-env.d.ts

### ISSUE THREAD

**Issue by JanStevens** [sst/ion#336]
Hi,

Since everyone has a different preference regarding formatting and code style it would be better of the generated `sst-env.d.ts` file adds the following to the top of the file.

```
/* tslint:disable */
/* eslint-disable */
```

Right now we have every sst deploy that changes the file so our PR linter can then complain about wrong format being used (we use `'` instead of `"`) 🙈 

Regards

**Comment by thdxr** [comment]
i don't love the idea of having to maintain a potentially ever growing list here. can you ignore it with a pattern in your eslint config? i'll probably end up adding it just so it works for everyone out of the box but wish there was a better way

**Comment by thdxr** [comment]
fixed in v0.0.341

**Comment by JanStevens** [comment]
@thdxr I think you made a small error and didn't account for newlines since the generated file now looks like

```typescript
/* tslint:disable *//* eslint-disable */import "sst"
declare module "sst" {
  export interface Resource {
    Account: {
      name: string
      type: "sst.aws.Dynamo"
    }
    Loyalty: {
      name: string
      type: "sst.aws.Dynamo"
    }
  }
}
export {}
```

Notice how they are all on one line and the `import 'sst'` is now also commented out
